torch/_numpy/__init__.py

```diff
@@ -0,0 +1,28 @@
+from . import fft, linalg, random
```

Comment:
Do you think it's useful to add a README.md under `torch/_numpy/`?

Reply:
Yep, that's in the TODO list in the OP